mailipy/gen.py

```diff
@@ -171,7 +172,7 @@ def render_template(template, data):
 
         msg["Subject"] = config["subject"]
         msg["Date"] = email.utils.formatdate()
-        msg["Message-Id"] = config["msgid"] % (datetime.datetime.now().strftime("%s") + str(random.random()))
+        msg["Message-Id"] = config["msgid"] % (datetime.datetime.now().strftime("%S") + str(random.random()))
```

Comment:
It seems that %s and %S aren't the same thing... This is what I get on my system:

```
>>> datetime.datetime.now().strftime("%S")
'30'
>>> datetime.datetime.now().strftime("%s")
'1666476693'
```

Maybe we can use this instead, which seems like it has the behavior we want:

```
>>> int(datetime.datetime.timestamp(datetime.datetime.now()))
1666476818
```

Reply:
>>> import datetime
>>> datetime.datetime.now().strftime("%S")
'26'
>>> datetime.datetime.now().strftime("%s")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: Invalid format string
>>>

and I can't fund the %s  desc in python doc ,,.. maybe

Reply:
Yes %s is not standard, we shoul replace it. I think the correct thing here is:

```
str(int(datetime.datetime.timestamp(datetime.datetime.now())))
```

Reply:
ok,  
str(int(datetime.datetime.timestamp(datetime.datetime.now())))
works for me